integration-test/src/main/java/com/sequenceiq/it/cloudbreak/RecommendationsTests.java

```diff
@@ -66,14 +65,14 @@ private String getJsonFile() throws IOException {
 
     private void createClusterDefinition() throws Exception {
         given(CloudbreakClient.created());
-        given(ClusterDefinition.request().withName(VALID_CD_NAME).withDescription(CD_DESCRIPTION).withClusterDefinition(getClusterDefinitionFile()));
-        when(ClusterDefinition.post());
+//        given(ClusterDefinition.request().withName(VALID_CD_NAME).withDescription(CD_DESCRIPTION).withClusterDefinition(getClusterDefinitionFile()));
```

Comment:
what will be the fate of this test class? who will rewrite it with the new framework?

Reply:
I don't know. I've created a task for it: https://jira.cloudera.com/browse/CB-615